src/main/java/com/robin/hoteless/Reservation.java

```diff
@@ -0,0 +1,42 @@
+package com.robin.hoteless;
+
+import java.time.LocalDate;
+import java.util.Date;
+
+public class Reservation {
+    int id;
+    Date made;
```

Comment:
This variable name might be better with the word date or time included. Outside of definition (this line), would you be able to identify what this variable is? Consider the example which would be rather confusing:

...
bedType = Types.KING;
made = "once per day?";
...